capstone-backend/src/test/java/com/google/sps/data/BookListTest.java

```diff
@@ -83,26 +85,27 @@ public void testSetters() {
     bookList2.setCollaboratorsIDs(COLLABORATORS_IDs_2);
     bookList2.setID(ID_2);
 
-    assertEquals(BOOKSHELF_ID_1, bookList2.getBookshelfID());
-    assertEquals(USER_ID_1, bookList2.getUserID());
+    assertEquals(true, bookList2.getUserID().equals(USER_ID_1));
+    assertEquals(true, bookList2.getBookshelfID().equals(BOOKSHELF_ID_1));
     assertEquals(COLLABORATORS_IDs_2, bookList2.getCollaboratorsIDs());
-    assertEquals(ID_2, bookList2.getId());
+    assertEquals(true, bookList2.getCollaboratorsIDs().containsAll(COLLABORATORS_IDs_2));
+    assertEquals(true, bookList2.getId().equals(ID_2));
   }
 
   @Test
-  public void testAddRemoveContainsCollaborators(){
+  public void testAddRemoveContainsCollaborators() {
 
-    bookList1.addCollaborator(COLLABORATOR_ID_2);
-    assertEquals(true, bookList1.containsCollaborator(COLLABORATOR_ID_2));
+    bookList1.addCollaborator(COLLABORATOR_ID_1);
+    assertEquals(true, bookList1.containsCollaborator(COLLABORATOR_ID_1));
 
-    bookList2.addCollaborator(COLLABORATOR_ID_1);
-    assertEquals(true, bookList2.containsCollaborator(COLLABORATOR_ID_1));
+    bookList2.addCollaborator(COLLABORATOR_ID_2);
+    assertEquals(true, bookList2.containsCollaborator(COLLABORATOR_ID_2));
 
-    bookList1.removeCollaborator(COLLABORATOR_ID_2);
-    assertEquals(false, bookList1.containsCollaborator(COLLABORATOR_ID_2));
+    bookList1.removeCollaborator(COLLABORATOR_ID_1);
+    assertEquals(false, bookList1.containsCollaborator(COLLABORATOR_ID_1));
```

Comment:
Replace this and similar statements with assertFalse(bookList1.containsCollaborator(COLLABORATOR_ID_1))